### DISCUSSION TITLE
More icons for Indicator panel

### DISCUSSION THREAD

**Discussion by rgov** [foxglove/community#458] [Q&A]
It would be a nice upgrade to the Indicator panel to be able to select icons beyond the "bulb". MUI gives you [a bunch of them](https://mui.com/material-ui/material-icons/) or the user could select their own icon files.

**Comment by jtbandes** [comment]
Have you experimented with putting emoji in the label?

**Comment by rgov** [comment]
I considered it but it seems like there are not too many relevant icons, and the aesthetics are a bit of a miss, but maybe [Noto](https://developers.googleblog.com/2022/04/what-is-black-and-white-and-read-all.html) might be suitable?
